### PR TITLE
refactor: remove ring dirs and update loops 

### LINKED ISSUE #6: Rings should not be determined by filepath
Right now, the Radar's import & export scripts use filepath, eg `radars/primary/quadrants/technologies/rings/adopted/blips/django.json`, to read what ring a blip is part of (see https://github.com/openedx/openedx-tech-radar/blob/main/scripts/export.js#L63-L65).

Rewrite these import/export scripts, and adjust all the filepaths, to not rely on filepath for ring definition. Why? Having the ring in the filepath means that for any needed change to a ring, a file must be moved. This destroys history and makes `git blame` worthless. It is also duplicative work, as the ring position is also defined in the json.

Suggestion: I suggest the filepaths should be, eg, `radars/primary/quadrants/technologies/blips/django.json`. That is, the _quadrant_ remains defined in the filepath. Why? We don't expect a blip's quadrant to change, and this makes the ~100+ blip definitions a little more easy to find, as they're sorted by quadrant.

### PR DESCRIPTION
**Pull request by Carlos-Muniz**
The <blip-name>.json files have been moved from within the
`radars/primary/` directory from:
`quadrants/<quadrant-name>/rings/<ring-name>/blips/<blip-name>.json` to
`quadrants/<quadrant-name>/blips/<blip-name>.json` so that when we move
a blip from ring to ring, the git blame is no longer worthless.

In addition, the forEach loops within `scripts/export.js` have been
updated to work with the newly refactored directories. And the ring
directory has been removed from being created in `scripts/import.js`.

Closes #6

### CODE REVIEW

File: scripts/export.js [hunk 1]
```diff
@@ -61,22 +52,14 @@ try {
...
-        blips.sort();
-        blips.forEach(blip => {
-          const blipPath = path.resolve(blipsPath, blip);
-          const blipData = require(blipPath);
-          radarJson.push(blipData);
-        });
+      const blipsPath = path.resolve(quadrantsPath, quadrant, 'blips');
+      const blips = fs.readdirSync(blipsPath);
+      // This will sort the blips alphabetically, regardless of ring.
+      blips.sort();
```
Comment (davidjoy) [inline scripts/export.js:58]: Hmm.  I think there's likely a possible bug here.  The order of rings in the tech radar visualization is based on their appearance in the CSV.  If the first blip alphabetically happens to be "Provisional", for example, that will make Provisional the innermost ring.  Adding blips by ring had the benefit of ensuring that the rings were added in the right order.

The ring information is in the blips themselves - I think we could add a compare function to the sort here and use an object like `ringSortValues` to ensure that the blips are ordered first by ring and _then_ alphabetically.  

Something like:

```
const ringSortValues = {
  adopted: 0,
  accepted: 1,
  provisional: 2,
  hold: 3,
};

blips.sort((a, b) => ringSortValues[a.ring] - ringSortValues[b.ring]);
```

That'll preserve the right behavior, I think.  Note: I didn't test that code.

**Review by davidjoy — CHANGES_REQUESTED** [review]
I think there's probably a little bug in there.  I suggested a fix.

File: scripts/export.js (hunk 1 above)
Comment (Carlos-Muniz) [reply scripts/export.js:58]: Should they be ordered first by ring, and then alphabetically by blip name? Or should they be ordered by quadrant as well? If so, how?

File: scripts/export.js [hunk 2]
```diff
@@ -14,22 +14,21 @@ const optionDefinitions = [
 
 const options = commandLineArgs(optionDefinitions);
 
-// These 'sort values' are used in a sort function to order our rings and quadrants correctly.
-// Lower values should be shown first.
```
[the remark this answers is not in the document]
Comment (Carlos-Muniz) [reply scripts/export.js:18]: This comment has been re-added.

**Review by davidjoy — APPROVED** [review]

**Pull request merged**